gems/aws-sdk-core/spec/aws/plugins/sign_spec.rb

```diff
@@ -222,10 +222,23 @@ module Plugins
         end
 
         it 'raises an error when attempting to sign a request w/out a token' do
-          client = TestClient.new(client_options.merge(token_provider: nil) )
-          expect {
+          client = TestClient.new(client_options.merge(token_provider: nil))
+          expect do
             client.operation
-          }.to raise_error(Errors::MissingBearerTokenError)
+          end.to raise_error(Errors::MissingBearerTokenError)
+        end
+      end
+
+      context 'sigv2' do
```

Comment:
This generic test is good - but it seems like we should have a test in S3 specifically for this as well

Reply:
The skipping is in Sign - this Client does not have sigv2 plugin added.